code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -829,23 +829,15 @@
 		return
 
 	var/mob/living/carbon/human/H = user
-	var/random = rand(1,4)
+	var/random = rand(1,2)
 
 	switch(random)
 		if(1)
-			to_chat(user, "<span class='danger'>Your appearance morphs to that of a very small humanoid ash dragon! You get to look like a freak without the cool abilities.</span>")
-			H.dna.features = list("mcolor" = "A02720", "tail_lizard" = "Dark Tiger", "tail_human" = "None", "snout" = "Sharp", "horns" = "Curled", "ears" = "None", "wings" = "None", "frills" = "None", "spines" = "Long", "body_markings" = "Dark Tiger Body", "legs" = "Digitigrade Legs")
-			H.eye_color = "fee5a3"
-			H.set_species(/datum/species/lizard)
-		if(2)
-			to_chat(user, "<span class='danger'>Your flesh begins to melt! Miraculously, you seem fine otherwise.</span>")
-			H.set_species(/datum/species/skeleton)
```

Comment:
But this is the best buff you can get out of the three...

Reply:
How can being a skeleton, looking ugly, being fragile to brute and not being able to heal normally is good..?

Reply:
Okay but seriously how the fuck is this a punishment, skeletons are overall superior to humans due to the advantages they have

Reply:
Their only advantage is literally bsing spaceproof as far as I know, plus if people had a choice to play a skeleton I'm sure a huge majority of our players wouldn't want to be one. Let's face it people do want to be their own characters and when they get turned into another character they aren't happy, at least that's how majority of our playerbase is. If you can prove me wrong then I am willing to be proven wrong, really..

Reply:
Skeles are no long space proof
Their are rad proof still tho!

Reply:
even more the reason to show me hoe come skeletons are so much more superior if that's true..